src/register.js

```diff
@@ -0,0 +1,56 @@
+const { v4 } = require("uuid");
+const AWS = require("aws-sdk");
+const { response, isValidEmail } = require("./utils");
+const bcrypt = require("bcryptjs");
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+
+const register = async (event) => {
+  const userId = v4();
+  try {
+    const { email, password } = JSON.parse(event.body);
+    if (!email || !password) {
+      return response(401, { message: "Email or password is empty" });
+    }
+
+    if (!isValidEmail(email)) {
+      return response(401, { message: "Invalid email" });
+    }
+
+    const params = {
+      TableName: "TodoTable",
```

Comment:
Store the name of the table into a variable that you define outside of lambda function